packages/@aws-cdk/aws-ec2/lib/vpc-peering-connection.ts

```diff
@@ -0,0 +1,137 @@
+import { Construct, IResource, Resource, Stack } from "@aws-cdk/core";
+import { CfnVPCPeeringConnection } from "./ec2.generated";
+import { IRoute, IVpc } from "./vpc";
+
+/**
+ * Options to add vpc peering conection to vpc
+ */
+export interface VpcPeeringConnectionOptions {
+  /**
+   * Peered Vpc
+   * Required: either a peeredVpcId or peeredVpc
+   * @default null
```

Comment:
Document default **behavior**, not default the **value**.

If I know JavaScript I know the value will be `undefined` (not `null`), but what does that mean to me as a user?

This comment applies to all occurrences of `@default null` in this PR.

Reply:
Not sure what I can write as default in this case where either peerVpc or peerVpcId is required, same goes for the routeType and routeId and the cidr/cidrIPv6, maybe some hints on how to write it would be useful

Reply:
In this particular case, I would put the "required" statement in the `@default` annotation, something like:

```
@default - Specify exactly one of peerVpc or peerVpcId
```

For other `@default` annotations, see what makes sense.